web/themes/custom/nfa/nfa.theme

```diff
@@ -34,6 +34,16 @@ function nfa_preprocess_node(&$variables) {
   }
 }
 
+/**
+ * Implements template_preprocess_page().
+ */
+function nfa_preprocess_page__tree_farmer_overview__sub_areas(&$variables) {
+  // Display the sub area map aon the sub areas page.
+  $variables['map_block'] = \Drupal::service('plugin.manager.block')
```

Comment:
Depending if you plan to add more like these, you could use https://www.drupal.org/project/twig_tweak and use {drupal_block} directly on twig

Reply:
I will most likely be adding more blocks so I will switch to Twig tweak next time. Thanks!